src/api/teams.iced

```diff
@@ -59,6 +59,23 @@ teams.get_all_teams = (callback) ->
   out = _.flatten(resps, true)
   return callback(null, out)
 
+teams.getTeamRolesForUser = (user, callback) ->
+  ###
+  return an array of team/role hashes to which the user belongs:
+    [{team: <obj>, role: <str>}]
+  ###
+  await teams.get_all_teams(defer(err, all_teams))
+  if err then return callback(err)
+
+  team_roles = []
+
+  for team in all_teams
```

Comment:
This is actually a horrible hack. I'm embarrassed it made it in the final. This should be a couchdb view. I have added an issue to replace this as soon as possible.